src/libs/actions/Session.js

```diff
@@ -80,7 +81,7 @@ function signOut() {
  * @param {String} login
  */
 function fetchAccountDetails(login) {
-    Onyx.merge(ONYXKEYS.ACCOUNT, {error: '', loading: true});
+    Onyx.set(ONYXKEYS.ACCOUNT, {error: '', loading: true});
```

Comment:
I saw `account` state include my email on local storage while I tried to sign in after logout. This only happened twice and it was creating the issue.

Reply:
Thanks for the description. Is there some clear way to reproduce what you experienced? 

And do you have any idea why the `account.email` persisted after you signed out? I'd think if we find out how this can happen then the fix would be to try to _prevent it_. And not only to clear the data before fetching the account details. Does that sound OK?

Reply:
I could not reproduce the issue with the steps you specified. I then tried over and over again. I logged in, wrote something in chat, and logged out. I don't have any idea why the account.email persisted after signing out. But on the contrary, when I persist my 'account.email' to local storage with the browser console, I saw happened this issue. Thats why defined with Onyx.set() in case any cookie value is entered incorrectly in the initial state of the account.

Reply:
Ok thanks for the explanation. I don't think we should merge the fix unless we understand how/why this can happen. It should not be necessary to replace the entire `account` key at the stage where we are doing this.